src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -4765,7 +4765,7 @@ paths:
               schema:
```

Comment:
What do you think about using `default:` for the error responses? It can save a lot of boilerplate in the yaml since all error responses have the same schema.

https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/main/resources/api/data-repository-openapi.yaml#L4482

https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/main/resources/api/data-repository-openapi.yaml#L3772

Reply:
With the default, we would lose the custom error messages, right? (like "Unauthorized - snapshot does not exist or missing permissions to view.")
 I think the [approach under "reusing responses" ](https://swagger.io/docs/specification/describing-responses/) is interesting, as that would remove a lot of the duplicated code without having to lose the more detailed error message.

Reply:
I started digging into this work and found that the error descriptions weren't really accurate and that our error handling could use some improvements. I started on this work but it's a bigger change, so I pulled this into a separate ticket. I hope we can discuss today. https://broadworkbench.atlassian.net/browse/DC-1058